onnx_transformers/pipelines.py

```diff
@@ -1722,10 +1750,13 @@ def pipeline(
 
     modelcard = None
     # Try to infer modelcard from model or config name (if provided as str)
-    if isinstance(model, str):
-        modelcard = model
-    elif isinstance(config, str):
-        modelcard = config
+    # TODO: Disable modelcard (below 4 lines) if working with local models.
+    # searches modelcard.json
+    if not local_model:
```

Comment:
we can keep it as it is, does it break for local model ? if not let's keep it as it is